### PR TITLE
Added system started flow trigger and dutch translations

### PR DESCRIPTION
**Pull request by irritanterik**

### CODE REVIEW

**Comment by irritanterik** [comment]
Hope you like the idea for a system flow cards section

**Comment by DieterKoblenz** [comment]
+1 for this pull request. Seems to work without problems.

File: lib/systemFlows.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  Homey.manager('flow').trigger('systemInitiated', null, null)
+  Homey.manager('flow').on('condition.systemCloudConnected', onConditiononSystemCloudConnected)
+  Homey.manager('flow').on('condition.systemFlowTimeout', onConditiononSystemFlowTimeout)
+  Homey.manager('flow').on('condition.systemFlowTimeout.units.autocomplete', onConditionSystemFlowTimeoutUnitsAutocomplete)
+}
+
+function onConditiononSystemCloudConnected (callback, args) {
+  getCloudStatus().then(function(result) {
+    callback(null, result.cloud_connected)
+  })
```
Comment (matjaz) [inline lib/systemFlows.js]: please add `catch` so callback is always called.

File: lib/systemFlows.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    {id: 'minutes', name: __("timeUnits.minutes")},
+    {id: 'hours', name: __("timeUnits.hours")},
+    {id: 'days', name: __("timeUnits.days")},
+    {id: 'weeks', name: __("timeUnits.weeks")},
+    {id: 'months', name: __("timeUnits.months")},
+    {id: 'years', name: __("timeUnits.years")},
+  ]
+
+  callback(null, list.filter(
+    item => item.name.toLowerCase().includes(args.query.toLowerCase())).map(
```
Comment (matjaz) [inline lib/systemFlows.js]: this indentation is confusing. please re-indend or create vars.

File: lib/systemFlows.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  flowTimeoutConditionTimestamps[args.units.marker] = new Date().getTime()
+  callback(null, true)
+}
+
+function timeDiffNow (start, units) {
+  var millisDiff = new Date().getTime() - start
+  switch(units) {
+    case 'seconds':
+      return millisDiff / 1000
+      break;
```
Comment (matjaz) [inline lib/systemFlows.js]: This app uses http://standardjs.com code style. please validate

File: lib/systemFlows.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    {id: 'hours', name: __("timeUnits.hours")},
+    {id: 'days', name: __("timeUnits.days")},
+    {id: 'weeks', name: __("timeUnits.weeks")},
+    {id: 'months', name: __("timeUnits.months")},
+    {id: 'years', name: __("timeUnits.years")},
+  ]
+
+  callback(null, list.filter(
+    item => item.name.toLowerCase().includes(args.query.toLowerCase())).map(
+      item => ({id: item.id, name: item.name, marker: new Date().getTime()})
```
Comment (matjaz) [inline lib/systemFlows.js]: nice marker workaround. :) you could move getTime call out of loop.

**Comment by matjaz** [comment]
Nice contribution! I've added few comments, please take a look and give me feedback.
Glad to merge it.

Thank you!

**Comment by irritanterik** [comment]
Improved on all your remarks. Except removing getTime() from the marker. I just like it that way (full numeric instead of date notation).

File: lib/systemFlows.js (hunk 2 above)
Comment (irritanterik) [reply lib/systemFlows.js]: Added var and change indentation

File: README.md [hunk 5]
```diff
@@ -7,7 +7,7 @@ Homey productivity boost.
 
 ## Condition cards
 - Cloud connected
-- Check when has run
+- Check when has run (not survices reboot)
```
Comment (matjaz) [inline README.md]: typo?

**Comment by matjaz** [comment]
LGTM. will test, merge and publish later today. Thank you!

File: README.md (hunk 5 above)
Comment (irritanterik) [reply README.md]: :(. Can correct it tonight.

Comment (irritanterik) [reply README.md]: Fixed

File: lib/systemFlows.js [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+    case 'hours':
+      return millisDiff / 1000 / 60 / 60
+    case 'days':
+      return millisDiff / 1000 / 60 / 60 / 24
+    case 'weeks':
+      return millisDiff / 1000 / 60 / 60 / 24 / 7
+    case 'months':
+      return millisDiff / 1000 / 60 / 60 / 12 / 365
+    case 'years':
+      return millisDiff / 1000 / 60 / 60 / 24 / 356
```
Comment (matjaz) [inline lib/systemFlows.js]: missed that one before. this looks wrong or am i missing something? same for months.

**Comment by matjaz** [comment]
I've submitted new version. Thank you!

**Pull request merged**